draft-ietf-privacypass-auth-scheme.md

```diff
@@ -112,6 +112,13 @@ from a specific issuer ({{challenge}}). Once a client has received a token
 from that issuer, or already has a valid token available, it presents the
 token to the origin ({{redemption}}).
 
+Unlike many authentication schemes in which a client will present the same credentials
+across multiple requests, tokens used with the "PrivateToken" scheme are generally
+single-use credentials, and are not reused. Specific token issuance protocols MAY
```

Comment:
I think I understand the intent here -- to allow issuance protocols where tokens can be "presented" more than once without privacy regressions -- but this text seems like it would allow clients to literally double spend the same value. That would be a privacy regression. Instead, I would phrase this as something like "spending the same token value more than once allows the origin to link multiple sessions to the same client," and omit mention of what future protocols may do.

Reply:
Looks good!